docs/source/serving/spot-policy.rst

```diff
@@ -0,0 +1,140 @@
+.. _spot_policy:
+
+Using Spot Instances
+====================
+
+SkyServe supports serving models on a mixture of spot and on-demand replicas with two options: :code:`base_ondemand_fallback_replicas` and :code:`dynamic_ondemand_fallback`.
+
+:code:`base_ondemand_fallback_replicas` sets the number of on-demand replicas to keep running at all times. This is useful for ensuring service availability and making sure that there is always some capacity available, even if spot replicas are not available. :code:`use_spot` should be set to :code:`true` to enable spot replicas.
+
+.. code-block:: yaml
+
+    service:
+        readiness_probe: /health
+        replica_policy:
+            min_replicas: 2
+            max_replicas: 3
+            target_qps_per_replica: 1
+            base_ondemand_fallback_replicas: 1
+
+    resources:
+        ports: 8081
+        cpus: 2+
+        use_spot: true
+
+    workdir: examples/serve/http_server
+
+    run: python3 server.py
+
```

Comment:
Can we add a tip/note for Kubernetes, noting that Kubernetes pods are counted as on-demand replicas and that this policy can do a mix of Kubernetes + cloud spot?